core/store/migrate/migrate.go

```diff
@@ -88,7 +87,7 @@ func ensureMigrated(db *sql.DB, lggr logger.Logger) {
 				}
 			}
 
-			if _, err = db.Exec(sql, id); err != nil {
+			if _, err = db.Exec(`INSERT INTO ? (version_id, is_applied) VALUES (?, true)`, goose.TableName(), id); err != nil {
```

Comment:
@jmank88 I made a fix in 3f50333e2f8bdee1d3417564c74cbe499cc2a90f but wanted to call this to your attention

Reply:
If it works :+1:

Reply:
How can we confirm?

Reply:
If CI passes then we're good :ok: